MekHQ/src/mekhq/gui/dialog/MekHQUnitSelectorDialog.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2020-2025 The MegaMek Team. All Rights Reserved.
+ * Copyright (C) 2025 The MegaMek Team. All Rights Reserved.
  *
  * This file is part of MekHQ.
```

Comment:
Don't remove copyright years. Earliest should be 2020 or first year file was created.